app/views/podcast_planner/_form_date_calculator.html.erb

```diff
@@ -0,0 +1,83 @@
+<section>
+  <div class="mb-4">
+    <h2 class="fw-bold">Date Calculation of Draft Batch</h2>
+    <p>
+      Calculate release dates for your drafts by selecting which days and weeks you plan to publish, and then select a date range.
+    </p>
+  </div>
+
+  <div class="mb-4">
+    <h3 class="fw-bold">Day selection</h3>
+    <p>Which day of the week do you plan to publish your episodes? Select all that apply.</p>
+    <div class="col-lg-12">
+      <div class="form-floating">
+        <%= form.select :selected_days, day_options, {}, data: {action: "podcast-planner#preview"}, multiple: true %>
+        <%= form.label :selected_days %>
+      </div>
+    </div>
+  </div>
+
+  <div class="mb-4" data-controller="toggle-field">
+    <h3 class="fw-bold">Week selection</h3>
+    <div class="mb-4">
+      <p>Which weeks of each month do you plan to publish? Select all that apply.</p>
+      <div class="col-lg-12" data-toggle-field-target="field" field="periodic">
+        <div class="form-floating">
+          <%= form.select :period, periodic_weeks_options, {}, data: {action: "podcast-planner#preview"} %>
```

Comment:
so i've added all of these `onChange` actions to all the inputs in this partial, so that the preview updates with changes on any of the inputs.